package com.github.wz2cool.dynamic.exception;

/**
 * @author Frank
 */
public class InternalRuntimeException extends RuntimeException {
    public InternalRuntimeException(Throwable cause) {
        super(cause);
    }
}
